crates/downloader/src/main.rs

```diff
@@ -458,6 +481,20 @@ fn octocrab() -> octocrab::Result<Arc<Octocrab>> {
     octocrab.build().map(Arc::new)
 }
 
+fn retry_result<T, E, Fun>(f: Fun, retries: u32) -> Result<T, E>
+where
+    Fun: Fn() -> Result<T, E>,
+{
+    let mut ctr = retries + 1;
+    while ctr > 1 {
+        if let Ok(o) = f() {
+            return Ok(o);
+        }
+        ctr -= 1;
+    }
+    return f();
```

Comment:
```suggestion
    for _ in 0..retries {
        if let Ok(o) = f() {
            return Ok(o);
        }
    }
    f()
```
これで良さそう。（特に最後はclippyでも警告するはず…だっけ？自信なさげ）